client/state/ui/olark/actions.js

```diff
@@ -7,7 +7,8 @@ import {
 	OLARK_REQUEST,
 	OLARK_TIMEOUT,
 	OLARK_OPERATORS_AVAILABLE,
-	OLARK_OPERATORS_AWAY
+	OLARK_OPERATORS_AWAY,
+	OLARK_SET_AVAILABILITY,
```

Comment:
Is the final comma here called for? I note that before this change, we didn't have one. Not sure if it's an issue or not, but thought I'd mention it just in case.